shavar.ini

```diff
@@ -26,23 +26,35 @@ pythonpath = local/bin
 paste-logger = %p
 
 [loggers]
-keys = root
+keys = root, sentry
 
 [handlers]
-keys = console
+keys = console, sentry
 
 [formatters]
 keys = generic
 
 [logger_root]
 level = INFO
-handlers = console
+handlers = console, sentry
+
+[logger_sentry]
+level=WARN
+handlers=console
+qualname = sentry.errors
+propagate = 0
 
 [handler_console]
 class = StreamHandler
 args = (sys.stdout,)
 formatter = generic
 
+[handler_sentry]
+class = sentry_sdk.integrations.logging.EventHandler
+args = ('WARNING',)
+formatter = generic
+dsn = ${SENTRY_DSN}
```

Comment:
When the `SENTRY_DSN` environment variable is not set I get the following error:
`configparser.InterpolationMissingOptionError: Bad value substitution: option 'dsn' in section 'handler_sentry' contains an interpolation key 'SENTRY_DSN' which is not a valid option name. Raw value: '${SENTRY_DSN}'`
Commenting out the line allows the app to run normally. Should I just comment it out?

Reply:
@skim1102 and I talked about this outside of GitHub. The configuration software doesn't appear to have the "value if not set" ability, like ERB or Bash, so we're setting ``SENTRY_DSN`` to an empty string when we start the docker container..